packages/stryker-webpack-transpiler/test/helpers/sinonInit.ts

```diff
@@ -0,0 +1,17 @@
+import * as sinon from 'sinon';
+import * as log4js from 'log4js';
+
+beforeEach(() => {
+  global.sandbox = sinon.sandbox.create();
+  global.logMock = {
+    debug: sandbox.stub(),
+    info: sandbox.stub(),
+    warn: sandbox.stub(),
+    error: sandbox.stub()
+  };
+  sandbox.stub(log4js, 'getLogger').returns(global.logMock);
+});
+
+afterEach(() => {
+  global.sandbox.restore();
```

Comment:
Can be shortened by doing `afterEach(() => global.sandbox.restore())`

Reply:
I know, but I've burned myself too often on this:

Lets say you have this line of code:

```ts
beforeEach(() => thing.initialize());
```

Later on you decide to add a statement. That is a pain already, since you need to add `{ ...; }`:

```ts
beforeEach(() => {
  thing = new Thing();
  thing.initialize();
});
```

Do you see the problem here? Since the result of `initialize()` is no longer (implicitly) returned, the test will run before the `initiliaze` is finished. 

This is why I decided to always use brackets when dealing with `beforeEach()`, `it()` and `afterEach()`

Reply:
Ah that's fair.